connections/src/app/shared/trigger/trigger-landing/trigger-landing.component.html

```diff
@@ -1 +1,27 @@
 <p>trigger-landing works!</p>
+<div data-theme="duke" >
+<div class="relative m-2">
```

Comment:
@rorepopo Next time you come back to this page...

`data-theme="duke" `
you can move the theme, into the next available div... don't need extra divs cluttering up a page
`<div class="relative m-2" data-theme="duke">`